impeller/typographer/backends/skia/typographer_context_skia.cc

```diff
@@ -450,6 +473,21 @@ std::shared_ptr<GlyphAtlas> TypographerContextSkia::CreateGlyphAtlas(
   }
   atlas_context_skia.UpdateBitmap(bitmap);
 
+  // If the new atlas size is the same size as the previous texture, reuse the
```

Comment:
This was an optimization that was reverted because TextureMTL::SetContents is unsafe.